SecurityPkg/Library/Tpm2DeviceLibDTpm/Tpm2Ptp.h

```diff
@@ -0,0 +1,53 @@
+/** @file
+  PTP (Platform TPM Profile) CRB (Command Response Buffer) interface used by dTPM2.0 library.

```

Comment:
Capitalization consistency
```suggestion
  PTP (Platform TPM Profile) CRB (Command Response Buffer) interface used by DTPM2.0 library.

```